aikau/src/main/resources/alfresco/documentlibrary/AlfDocumentFilters.js

```diff
@@ -154,11 +153,11 @@ define(["dojo/_base/declare",
       /**
        * @instance
        */
-      onShowMoreClick: function alfresco_documentlibrary_AlfDocumentFilters__onShowMoreClick(evt) {
+      onShowMoreClick: function alfresco_documentlibrary_AlfDocumentFilters__onShowMoreClick() {
```

Comment:
When it's an inherited method, I annotate and when it's only used in the instance, I remove it as redundant. At least that was the intention.